rllib/tests/agents/functionality/test_pg_functionality.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import absolute_import
```

Comment:
How about we put this file in `agents/pg` to be close to the code? With bazel the tests can go anywhere as long as they have an entry in the BUILD file.

Reply:
As long as we do the location of tests consistently, it's ok with me. Will start cleaning this up.